templates/package/content/debian.tmpl

```diff
@@ -0,0 +1,65 @@
+{{if eq .PackageDescriptor.Package.Type "debian"}}
+	<h4 class="ui top attached header">{{.locale.Tr "packages.installation"}}</h4>
+	<div class="ui attached segment">
+		<div class="ui form">
+			<div class="field">
+				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.debian.registry"}}</label>
+				<div class="markup"><pre class="code-block"><code>sudo curl <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{$.PackageDescriptor.Owner.Name}}/debian/repository.key"></gitea-origin-url> -o /etc/apt/trusted.gpg.d/gitea-{{$.PackageDescriptor.Owner.Name}}.asc
+echo "deb <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{$.PackageDescriptor.Owner.Name}}/debian"></gitea-origin-url> &lt;distribution&gt; &lt;component&gt;" | sudo tee -a /etc/apt/sources.list.d/gitea.list
+sudo apt update</code></pre></div>
+				<p>{{.locale.Tr "packages.debian.registry.info" | Safe}}</p>
+			</div>
+			<div class="field">
+				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.debian.install"}}</label>
+				<div class="markup">
+					<pre class="code-block"><code>sudo apt install {{$.PackageDescriptor.Package.Name}}={{$.PackageDescriptor.Version.Version}}</code></pre>
+				</div>
+			</div>
+			<div class="field">
+				<label>{{.locale.Tr "packages.debian.documentation" | Safe}}</label>
+			</div>
+		</div>
+	</div>
+
+	<h4 class="ui top attached header">{{.locale.Tr "packages.debian.repository"}}</h4>
+	<div class="ui attached segment">
+		<table class="ui single line very basic table">
+			<tbody>
+				<tr>
+					<td class="collapsing"><h5>{{.locale.Tr "packages.debian.repository.distributions"}}</h5></td>
+					<td>{{Join .Distributions ", "}}</td>
+				</tr>
+				<tr>
+					<td class="collapsing"><h5>{{.locale.Tr "packages.debian.repository.components"}}</h5></td>
+					<td>{{Join .Components ", "}}</td>
+				</tr>
+				<tr>
+					<td class="collapsing"><h5>{{.locale.Tr "packages.debian.repository.architectures"}}</h5></td>
+					<td>{{Join .Architectures ", "}}</td>
```

Comment:
I am fairly certain that doesn't work anymore.

Reply:
The `Join` method is still there if you talk about #23982.

Reply:
I'm talking about https://github.com/go-gitea/gitea/commit/88201914767e15a51c910d295647d97973fa09bf.
But okay, `Join` was apparently not converted yet.

Reply:
But now ... this one #24417  😁

Reply:
I queued https://github.com/go-gitea/gitea/pull/24417 for merge.

Reply:
Didn't expect that I would ever say that, but apparently my review was "ahead of its time" :grin:

Reply:
Sorry for the inconvenience caused by my refactoring PRs, but I think I have to do so. 

There are too many technical debts, if we do not clean them up, there would be more bugs and waste more times.

If there is anything I can help, I can also help to fix conflicts caused by me and make new PRs work with these refactoring PRs.

Reply:
I absolutely agree that they are necessary.
Without them, Gitea would become pretty much unmaintainable over time.

Reply:
@wxiaoguang your cleanups are very welcome. I applaud especially the quality of them.